### PR TITLE
Redact multiconfig support

### PR DESCRIPTION
**Pull request by pangea-andrest**

### CODE REVIEW

File: packages/pangea-sdk/tests/integration/asyncio/test_authn.py [hunk 1]
```diff
@@ -279,12 +279,12 @@ async def authn_c2_login_n_client_session_invalidate(self):
             for session in response.result.sessions:
                 try:
                     await self.authn.client.session.invalidate(token=token, session_id=session.id)
-                except pexc.PangeaAPIException as e:
+                except pe.PangeaAPIException as e:
                     print(f"Fail to invalidate session_id[{session.id}] token[{token}]")
```
Comment (ggallien13) [inline packages/pangea-sdk/tests/integration/asyncio/test_authn.py:283]: This should be "Failed to invalidated..."

File: packages/pangea-sdk/tests/integration/asyncio/test_authn.py [hunk 2]
```diff
@@ -319,7 +319,7 @@ async def test_authn_z1_user_list(self):
         for user in response.result.users:
             try:
                 await self.authn.user.delete(email=user.email)
-            except pexc.PangeaAPIException:
+            except pe.PangeaAPIException:
                 print(f"Fail to delete user email: {user.email}")
```
Comment (ggallien13) [inline packages/pangea-sdk/tests/integration/asyncio/test_authn.py:323]: Same here, should be "Failed', instead of "Fail".

Comment (pangea-andrest) [reply packages/pangea-sdk/tests/integration/asyncio/test_authn.py]: fixed

File: packages/pangea-sdk/tests/integration/asyncio/test_authn.py (hunk 1 above)
Comment (pangea-andrest) [reply packages/pangea-sdk/tests/integration/asyncio/test_authn.py]: fixed

**Review by ggallien13 — APPROVED** [review]

**Pull request merged**